README.md

```diff
@@ -112,11 +112,23 @@ maskinen:
 
 - Legge inn readme for pakkene!
 
-## Intellj og stubs
+## Synkroniser TS-typer for med fpsak DTO-er
+
+Alle DTO-er fra upstream-tjenester (f.eks. fpsak) er samlet i én generert fil med TypeScript-typer.
+Synkroniser jevnlig for å holde typene oppdatert.
+
+- **Synkroniser typer i dev:**
+```bash
+./generate-types.sh
```

Comment:
kunne du lagt generate-types.sh under folder script? Og evt lagt eit script i package.json for å starta det?